sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -131,11 +131,12 @@ def read(
         # type: (...) -> Dict[str, Any]
         """Read the container properties.
 
+        :param list args: The explicit arguments of the method.
```

Comment:
We shouldn't actually document these - rather, we should document the specific args in the overload above.
If pylint requires that these be documented, this should be just typed as `Any`.